.rubocop.yml

```diff
@@ -0,0 +1,77 @@
+require:
+  - rubocop-rails
+  - rubocop-rspec
+
+AllCops:
+  TargetRubyVersion: 3.1.0
+  NewCops: enable
+  Exclude:
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'vendor/**/*'
+    - 'script/**/*'
+    - 'bin/**/*'
+    - 'node_modules/**/*'
+    - 'spec/vcr/**/*'
+    - 'spec/spec_helper.rb'
+    - 'spec/rails_helper.rb'
+
+Layout/LineLength:
+  Max: 120
+  Exclude:
+    - config/**/*
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/Alias:
+  Enabled: false
+
+Style/HashLikeCase:
```

Comment:
I'd enable this one - it's good practice to replace case-when instructions with hash.fetch. Sometimes it's even bonus points on recruitment tasks, depends on the company :)